util/PhotoMigration.java

```diff
@@ -0,0 +1,190 @@
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+    One time use utility to migrate file system photos
+      to the Photo table's respective clob field
+ */
+public class PhotoMigration {
+
+    public class Photo {
+        public String FilePath;
+        public int Id;
+    }
+
+    public static void main(String[] args) {
+
+        try
+        {
+            if(args.length >= 5)
+            {
+                String url  = args[0];
+                String user = args[1];
+                String pass = args[2];
+                String patPath = args[3];
+                String encPath = args[4];
+
+                PhotoMigration pm = new PhotoMigration(url, user, pass, patPath, encPath);
+
+                //NOTE: Set to TRUE if you want the migration
+                //       to overwrite existing blob fields
+                pm.MigratePhotos(false);
+            }
+            else
+            {
+                System.out.println("Usage: [db_url] [db_username] [db_password] [patient photo path] [encounter photo path]");
+            }
+        }
+        catch (Exception ex)
+        {
+            System.out.println(ex.getMessage());
+        }
+
+
+    }
+
+    private String _patPhotoPath;
+    private String _encPhotoPath;
+    private String _dbUrl;
+    private String _dbUser;
+    private String _dbPass;
+
+    public PhotoMigration(String dbUrl, String dbUserName, String dbPassword, String patientPhotoPath, String encounterPhotoPath)
+    {
+        _dbUrl = dbUrl;
```

Comment:
@b6025 what should i use for dbUrl? I tried:

localhost
and
jdbc:mysql://localhost/test?characterEncoding=UTF-8

Both say "No suitable driver found"